learningresources/models.py

```diff
@@ -61,6 +61,7 @@ class StaticAsset(BaseModel):
     Holds static assets for a course (css, html, javascript, images, etc)
     """
     course = models.ForeignKey(Course)
+    path = models.TextField()  # path in static folder
```

Comment:
Okay. I'll check it out. Incidentally, asset.name is an actual site; Gmail highlighted it and I clicked it without thinking.